File: programacao/c/respostas/selecao/07.c
#include <stdio.h>

int main(void) {
    char sexo;
    int idade;

    printf("Digite o sexo (M/F): ");
    scanf(" %c", &sexo);
    printf("Digite a idade: ");
    scanf("%d", &idade);

    if ((sexo == 'F' && idade > 45) || (idade < 18)) {
        printf("Tem direito a desconto\n");
    } else {
        printf("Nao tem direito a desconto\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/07.py
sexo = input("Digite o sexo (M/F): ")
idade = int(input("Digite a idade: "))

if (sexo == "F" and idade > 45) or (idade < 18):
    print("Tem direito a desconto")
else:
    print("Nao tem direito a desconto")
